neurobooth_os/tasks/task.py

```diff
@@ -361,6 +361,31 @@ def doDriftCorrect(self, vals):
         if self.eye_tracker is not None:
             self.eye_tracker.tk.doDriftCorrect(*vals)
 
+    def update_tablet_background(self, x: float, y: float) -> None:
+        '''
+           Draw a cross and a box on the eyelink tablet.
+           x and y are positions in 0 centered psychopy coordinate space.
+           x and y can be int or float
+           See pos_psych2pix above
+        '''
+
+        # First clear tablet screen
+        self.sendCommand('clear_screen 0')
+
+        # convert from 0 centered to top-left centered coordinate space
+        top_left_xy = self.pos_psych2pix([x, y])
+
+        # draw cross at top_left centered x,y position
+        self.sendCommand('draw_cross %d %d 10' % tuple(top_left_xy))
+
+        # draw box around cross
+        half_box_len_in_pix = 50 ## this is box_length of 100 pixels
+        # box coords are defined in a single list as x,y at the diagonal vertices
+        # color is defined by number - 12 is for red, 10 is for green
+        box_coords_top = [top_left_xy[0]-half_box_len_in_pix, top_left_xy[1]-half_box_len_in_pix]
+        box_coords_bot = [top_left_xy[0]+half_box_len_in_pix, top_left_xy[1]+half_box_len_in_pix]
+        self.sendCommand('draw_box %d %d %d %d 12' % tuple(box_coords_top + box_coords_bot))
```

Comment:
Similarly: we could define `tablet_draw_box` and have a map from valid colors to integers.

Reply:
Done - see method on hostpc class